### PR TITLE
[IT] Add `osu!api`

### PR DESCRIPTION
**Pull request by itsVxeto**
Translation of the page in Italian.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!api/it.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# osu!api
+
+osu! ha un'[API](https://en.wikipedia.org/wiki/API) pubblica che servizi di terze parti possono usare per espandere e supportare la piattaforma osu!. Una volta era privata, solo per essere utilizzata su richiesta, ma ora chiunque può utilizzare l'API fino a quando hanno un account osu!.
```
Comment (marcostudios) [inline wiki/osu!api/it.md:3]:
```suggestion
osu! ha un'[API](https://en.wikipedia.org/wiki/API) pubblica che servizi di terze parti possono usare per espandere e supportare la piattaforma osu!. Una volta era privata, utilizzabile solo su richiesta, ma ora chiunque può utilizzare l'API purchè si abbia un account osu!.
```

File: wiki/osu!api/it.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Puoi trovare la documentazione sulla [wiki di osu-api](https://github.com/ppy/osu-api/wiki). La versione corrente dell'API utilizza chiavi private per autenticare le richieste. Puoi ottenere la chiave dell'API nella [pagina dell'account](https://osu.ppy.sh/home/account/edit#legacy-api). Si noti che la documentazione sulla wiki dell'osu! api potrebbe non essere completamente completa o estesa, e l'API stessa non in fase di sviluppo attivo.
```
Comment (marcostudios) [inline wiki/osu!api/it.md:5]:
```suggestion
Puoi trovare la documentazione sulla [wiki di osu-api](https://github.com/ppy/osu-api/wiki). La versione corrente dell'API utilizza chiavi private per autenticare le richieste. Puoi ottenere la chiave dell'API nella [pagina delle impostazioni dell'account](https://osu.ppy.sh/home/account/edit#legacy-api). Si noti che la documentazione sulla wiki dell'osu! api potrebbe non essere completamente completa o estesa, e l'API stessa non è in fase di sviluppo attivo.
```

File: wiki/osu!api/it.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+---
+
+La versione 2 dell'osu!api è in lavorazione all'[osu-web](https://github.com/ppy/osu-web), e mira di essere più robusta dell'originale. La documentazione (non garantito per essere aggiornato) può essere trovata alla [pagina di riferimento](https://docs.ppy.sh).
```
Comment (marcostudios) [inline wiki/osu!api/it.md:9]:
```suggestion
La versione 2 dell'osu!api è in lavorazione all'[osu-web](https://github.com/ppy/osu-web), e mira ad essere più robusta dell'originale. La documentazione (non garantita per essere aggiornata) può essere trovata alla [pagina di riferimento](https://docs.ppy.sh).
```

File: wiki/osu!api/it.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+L'autenticazione per le nuove API seguono il [protocollo OAuth 2](https://oauth.net/2/). I client possono essere gestiti tramite la [pagina dell'account](https://osu.ppy.sh/home/account/edit#oauth).
```
Comment (marcostudios) [inline wiki/osu!api/it.md:11]:
```suggestion
L'autenticazione per le nuove API segue il [protocollo OAuth 2](https://oauth.net/2/). I client possono essere gestiti tramite la [pagina delle impostazioni dell'account](https://osu.ppy.sh/home/account/edit#oauth).
```

**Review by marcostudios — COMMENTED** [review]
These are my personal suggestions, feel free to add or discuss them.

**Review by cl8n — APPROVED** [review]

**Pull request merged**